apps/chef_objects/src/chef_objects.app.src

```diff
@@ -29,5 +29,71 @@
                   folsom,
                   pooler
                  ]},
-  {env, []}
+  {env, [
+         %% S3 Access credentials for Bookshelf operations
```

Comment:
I'm open to the idea, but that's not what this is about. I'm just trying to document what already exists.

I'm open to redoing configuration settings, but that is an annoying refactor, so it's best to have a large scale plan around it before starting.